examples/nextjs-app/app/api/oauth/github/route.ts

```diff
@@ -24,7 +24,7 @@ export const GET = async (request: Request) => {
 		const session = await auth.createSession(user.userId);
 		const authRequest = auth.handleRequest({ request, cookies });
 		authRequest.setSession(session);
-		return NextResponse.redirect(url.toString());
+		return NextResponse.redirect(new URL("/", url));
```

Comment:
Interested in why this was changed?

Reply:
I changed the code to Discord Provider, and the status code returned `500`.
I found that the server requested the `/api/oauth2/token` twice; the first was `200` and then got `400`.
Because the `code` has invalid.
I realize that it has been redirected to the same route.

Reply:
Yup, ok so it seems it only takes a `URL` object, weirrd